### DISCUSSION TITLE
Hello all!!

### DISCUSSION THREAD

**Discussion by cornzyblack** [OpenMined/courses#45] [Introduce Yourself]
Hello all, I am Eniwoke (E-knee-woe-keh) from Nigeria, I am really happy for the opportunity to learn more about privacy and to meet you all as well.

Please feel free to connect with me on LinkedIn [here](https://www.linkedin.com/in/eniwoke-c-b71852a1)

**Comment by yemikifouly** [comment]
Hi Eniwoke! 👋 I am Yemissi from Benin! 🙂 Welcome to the discussion board! I hope you enjoy the course. And feel free to comment on the posts of other students on the board. It is a great way to connect. 🙂

**Reply by cornzyblack** [reply]
Thanks
